examples/copyNumber.html

```diff
@@ -80,7 +80,7 @@
                         }
                     ]
                 };
-
+        options.locus=options.locus.toLowerCase();
```

Comment:
please move this to igv.createBrowser() in igv-create.js file. 
We don't want to do this lowercase change manually each time we initialize an igv browser.